MAINTAINERS.yml

```diff
@@ -3531,7 +3531,7 @@ Silabs Platforms:
     - dts/arm/silabs/
     - dts/bindings/*/silabs*
     - drivers/*/*_gecko*
-    - drivers/bluetooth/hci/slz_hci*
+    - drivers/bluetooth/hci/hci_silabs_*
```

Comment:
This change will likely cause a merge conflict with https://github.com/zephyrproject-rtos/zephyr/pull/80128 - nothing to be done about it until one of the PRs is merged, though, unless we roll the other PR into this one.